mutatex/utils.py

```diff
@@ -208,6 +207,14 @@ def parse_poslist_file(fname):
             log.error("the position list file is not in the right format")
             log.error("format error at %s" % line.strip())
             raise TypeError
+        if residue not in unique_residues:
+            pdb_residues_list=[]
+            for i in unique_residues:
+                single_set=set(i)
+                pdb_residues_list.append(set(residue).issubset(single_set))
```

Comment:
it's a bit more compact and easier to read if you squash these two lines together i.e.

```
pdb_residues_list.append(set(residue).issubset(set(i)))
```

Reply:
done